nemo/collections/asr/models/ctc_models.py

```diff
@@ -47,6 +48,13 @@ def list_available_models(cls) -> Optional[PretrainedModelInfo]:
         return result
 
     def __init__(self, cfg: DictConfig, trainer: Trainer = None):
+        # Get global rank and total number of GPU workers for IterableDataset partitioning, if applicable
+        self.global_rank = 0
+        self.world_size = 0
+        if trainer is not None:
+            self.global_rank = (trainer.node_rank * trainer.num_processes) + trainer.local_rank
+            self.world_size = trainer.num_nodes * trainer.num_processes
```

Comment:
Let's use trainer.num_gpus instead

Reply:
Just wondering--is there a significant difference here? The PTL trainer documentation that I've found hasn't been terribly illuminating, and only gives some details about `num_processes` while having very little about `num_gpus`.